core/test.sh

```diff
@@ -21,20 +21,6 @@ if [ -z "$CI" ]; then
     trap cleanup EXIT
 fi
 
-# Ensure each constant in `Constants.sol` is only defined once
-if [ "$CHECK_CONSTANTS" = true ]; then
-    constants=$(grep -oh '[_A-Z]* =' ./src/libraries/constants/Constants.sol)
-    while read constant; do 
-      n=$(grep -rnw --include=\*.sol '.' -e "$constant" | wc -l)
-      if (( n > 1)); then
-        echo "❌ `${constant}` is defined more than once" >> $GITHUB_STEP_SUMMARY
-        exit 1
-      fi
-    done <<< "$constants"
-
-    echo "✅ Each constant is only defined once" >> $GITHUB_STEP_SUMMARY
-fi
-
```

Comment:
Why are we removing this in this PR?

Reply:
The comment that I wrote for the `MANIPULATION_THRESHOLD_DIVISOR` constant was triggering weird false positives. I think it has something to do with `grep` matching on the `=` even though the `=` is inside the comment.

If you know how to fix it (instead of deleting) lmk.